build/release-tars/BUILD

```diff
@@ -217,11 +223,17 @@ release_filegroup(
     name = "release-tars",
     srcs = [
         ":kubernetes.tar.gz",
-        ":kubernetes-client-%s.tar.gz" % PLATFORM_ARCH_STRING,
-        ":kubernetes-node-%s.tar.gz" % PLATFORM_ARCH_STRING,
-        ":kubernetes-server-%s.tar.gz" % PLATFORM_ARCH_STRING,
-        ":kubernetes-manifests.tar.gz",
         ":kubernetes-src.tar.gz",
-        ":kubernetes-test.tar.gz",
     ],
+    conditioned_srcs = for_platforms(
+        for_all = [
+        ],
+        for_client = [":kubernetes-client-{OS}-{ARCH}.tar.gz"],
+        for_node = [":kubernetes-node-{OS}-{ARCH}.tar.gz"],
+        for_server = [
+            ":kubernetes-server-{OS}-{ARCH}.tar.gz",
+            ":kubernetes-manifests.tar.gz",
+        ],
+        for_test = [":kubernetes-test.tar.gz"],  # FIXME
```

Comment:
Can you explain how you want this to be fixed? (aka make platform specific?)

Reply:
done. changes are coming with https://github.com/kubernetes/enhancements/issues/714 but I didn't want to do any of that yet.